content/articles/domain-apex-heroku.markdown

```diff
@@ -1,33 +1,63 @@
 ---
 title: Pointing the Domain Apex to Heroku
-excerpt: Welcome to DNSimple. This page is about pointing the domain apex at Heroku. Hosted DNS has never been this easy.
+excerpt: How to point your domain to your Heroku application using DNSimple.
 categories:
 - Heroku and DNSimple
 ---
 
 # Pointing the Domain Apex to Heroku
 
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
 Your domain apex is the second or third level domain provided to you by your domain registrar. For example, `dnsimple.com` is the apex domain for DNSimple.
 
-To point your apex to Heroku you have 2 possibilities:
+To point your apex to Heroku you have 2 options:
 
-- Use the Heroku one-click service (for non-SSL usage)
-- Add an [ALIAS record](/articles/alias-record/) pointing your custom Heroku end point (for non-SSL and SSL usage)
+1. Using the Heroku one-click service.
+1. Manually adding an [ALIAS record](/articles/alias-record/).
 
-## Creating an ALIAS for the apex domain
 
-Go to the advanced editor page for your domain to manage your DNS records. Click *Add a Record* and select the *ALIAS* record type.
+Once your apex domain is pointing to correct place you will have to configure the custom domain at Heroku.
 
-![Add a Record](/files/dnsimple-alias-record.png)
+## Using the Heroku one-click service
 
-In the form enter the following settings:
+You can only use the one-click service if you are not using SSL on your Heroku application. If you are please follow the instructions in [the next section](#point-using-alias).
```

Comment:
We do have a [one-click Heroku SSL service](https://github.com/aetrion/dnsimple-services/blob/master/services/heroku-ssl/readme.md).

I'd keep the manual instructions, but we should mention that the user can use the right one-click service depending on whether he/she's using SSL or not.